src/pages/trader-tools/swap-calculator/_swap-calculator.tsx

```diff
@@ -241,19 +255,21 @@ const SwapCalculator = () => {
                 </SectionSubtitle>
 
                 <Flex mt="80px" mb="40px" tablet={{ mt: '40px', mb: '24px' }}>
-                    <SwapTabSelector
-                        active={tab === 'Synthetic'}
-                        onClick={() => onTabClick('Synthetic')}
-                    >
-                        <Text size="var(--text-size-m)" align="center">
-                            {localize('Synthetic')}
-                        </Text>
-                    </SwapTabSelector>
-                    <SwapTabSelector active={tab === 'Real'} onClick={() => onTabClick('Real')}>
-                        <Text size="var(--text-size-m)" align="center">
-                            {localize('Financial')}
-                        </Text>
-                    </SwapTabSelector>
+                    <ROW>
```

Comment:
based on [this PR](https://github.com/binary-com/deriv-com/pull/3480), we are moving towards conditional rendering, please use values from `useCountryRule` hook, thanks :pray: